.github/config.yml

```diff
@@ -23,8 +23,10 @@ newPRWelcomeComment: |
 firstPRMergeComment: |
  Congrats on merging your first pull request! 🙌🎉⚡️
  Your code will likely be published to PublicLab.org in the next few days, but first it will be published to https://stable.publiclab.org/ (it will take some minutes for this to load, and until then you may see logs from the build process). Please test out your work on this testing server and report back with a comment that all has gone well!
+ Do join our weekly check-in to share your this week goal and the awesome work you did 😃. Please find the link **pinned in the issue section** 📝
  Now that you've completed this, you can help someone else take their first step!
  Reach out to someone else working on theirs on [Public Lab's code welcome page](https://code.publiclab.org#r=all). Thanks!
+ Also please refer [here](https://github.com/publiclab/plots2#installation) for installation help 💿
```

Comment:
```suggestion
```

This will be suitable for the new PR welcome comment as this firstPRMergeComment will be shown after contributor will be done with the task.

Reply:
oh yeah :stuck_out_tongue: 
And i later discovered [this](https://github.com/publiclab/plots2/issues/3840) was thinking if this is a better link to add?